packages/@aws-cdk/aws-ec2/lib/nat.ts

```diff
@@ -6,6 +6,22 @@ import { Port } from './port';
 import { SecurityGroup } from './security-group';
 import { PrivateSubnet, PublicSubnet, RouterType, Vpc } from './vpc';
 
+/**
+ * Pair represents a gateway created by NAT Provider
+ */
+export interface IGateway {
```

Comment:
If the data conceptually represents a struct/record, we do not preprend with an `I`, so it would be `Gateway`. Maybe that's a little too generic, so I would maybe call it `GatewayConfig`.

Reply:
👍